src/renderer/components/search/CommandPalette.tsx

```
@@ -188,10 +210,20 @@ export const CommandPalette = (): React.JSX.Element | null => {
 
   // Fetch repository groups if needed
   useEffect(() => {
-    if (commandPaletteOpen && searchMode === 'projects' && repositoryGroups.length === 0) {
+    if (
+      commandPaletteOpen &&
+      (searchMode === 'projects' || globalSearchEnabled) &&
+      repositoryGroups.length === 0
+    ) {
       void fetchRepositoryGroups();
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)